src/components/Card/Card.js

```diff
@@ -1,12 +1,17 @@
 import './Card.css';
+import AOS from 'aos';
+import 'aos/dist/aos.css'; 
 
-const Card = ({ movieTitle, moviePoster }) => {
+
+AOS.init();
+
+const Card = ({ movieTitle, moviePoster, movieYear, id }) => {
   return (
-    <div className='movie-card'>
-      <img src={moviePoster} className='movie-poster' alt={movieTitle} />
-      <h3>
-        {movieTitle}
-      </h3>
+    <div className='movie-card' data-aos="fade-up">
+      <div className='movie-card-image-container'>
+        <img src={moviePoster} className='movie-poster' alt={movieTitle} />
+      </div>
+      <h3 className='movie-title'> {`${movieTitle} (${movieYear})`}</h3>
```

Comment:
Looks like we're missing an `<h1>` and `<h2>` tag, maybe we should modify this tag to `<h1>`?